packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -474,7 +474,7 @@ contract Bridge is EssentialContract, IBridge {
         ) {
             // For Taiko mainnet
             // 384 seconds = 6.4 minutes = one ethereum epoch
-            return (6 hours, 384 seconds);
+            return (1 hours, 384 seconds);
```

Comment:
1 hour after yesterday's discussion

Reply:
so long to return, can this improve codes

Reply:
Improve security.